modules/sdpolicy-am-compute.adoc

```diff
@@ -14,7 +14,7 @@ Multiple availability zone clusters require a minimum of 3 worker nodes for Cust
 All worker nodes within a single {product-title} machine pool must be of the same type and size. However, worker nodes across multiple machine pools within an {product-title} cluster can be of different types and sizes.
 ====
 
-Control plane and infrastructure nodes are also provided by Red Hat. There are at least 3 control plane nodes that handle etcd and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control plane and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes. See the Red Hat Operator support section below for more information about Red Hat workloads that must be deployed on worker nodes.
+Control plane and infrastructure nodes are also provided by Red{nbsp}Hat. There are at least 3 control plane nodes that handle etcd and API-related workloads. There are at least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. You must not run any workloads on the control plane and infrastructure nodes. Any workloads you intend to run must be deployed on worker nodes. See the Red Hat Operator support section below for more information about Red Hat workloads that must be deployed on worker nodes.
```

Comment:
Consider:

Red{nbsp} provides control plane and infrastructure nodes and these nodes are listed as follows:

* At least 3 control plane nodes that handle etcd and API-related workloads.
* At least 2 infrastructure nodes that handle metrics, routing, the web console, and other workloads. 

You must not run any workloads on the control plane and infrastructure nodes **[other workloads contradicts this statement?]**. Any workloads you intend to run must be deployed on worker nodes. 

Additional resources,

* For more information about running workloads that must be deployed on worker nodes, see [link].